modules/standard/Version.chpl

```diff
@@ -22,33 +22,85 @@
 
 .. highlight:: chapel
 
-This module contains features that support compile-time reasoning
-about version numbers in general, and Chapel version numbers
-specifically.  In more detail, it features:
+This module contains features that support reasoning about version numbers.
+The :type:`versionValue` supports compile-time reasoning about version numbers
+in general, and Chapel version numbers specifically, while the :type:`version`
+supports run-time reasoning about version numbers.
```

Comment:
This (or below when we talk about `version` and `versionValue` individually) might be an appropriate place to mention that we are not 100% compatible with actual semver spec.